code/modules/jobs/job_types/roguetown/adventurer/types/combat/warrior.dm

```diff
@@ -166,16 +179,85 @@
+			shoes = /obj/item/clothing/shoes/roguetown/boots/armoriron
+			belt = /obj/item/storage/belt/rogue/leather/black
+			neck = /obj/item/storage/belt/rogue/pouch/coins/mid
+			ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
+			ADD_TRAIT(H, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
+			H.dna.species.soundpack_m = new /datum/voicepack/male/knight()
+		//leper, darkest dungeon based. Sick, blind and rotting away but has massive strength if he can land it.
+		if("Leper")
+			H.set_blindness(0)
+			to_chat(H, span_warning("A noble, or a king of lands far away and now forgotten, broken and discarded. You roam the lands, strengthening your mind to find peace with your rotting husk, while taking punishments others can not."))
+			H.mind.adjust_skillrank_up_to(/datum/skill/combat/swords, 4, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/combat/wrestling, 2, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/combat/unarmed, 2, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/misc/athletics, 4, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/combat/knives, 2, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/misc/swimming, 1, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/misc/climbing, 2, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/misc/medicine, 3, TRUE)
+			H.mind.adjust_skillrank_up_to(/datum/skill/misc/reading, 5, TRUE) //poetry
+			//This is going to be crazy, horrible drawbacks, massive strength. Totally a bad idea.
+			H.change_stat("strength", 4)
+			H.change_stat("perception", -4) //you are going blind.
+			H.change_stat("endurance", 2)
+			H.change_stat("constitution", 4)
+			H.change_stat("speed", -4)
+			//You are not gonna be able to fight without your mask, keep it from breaking..
+			mask = /obj/item/clothing/mask/rogue/facemask/leper/gold
+			gloves = /obj/item/clothing/gloves/roguetown/leather/angle/leper
+			pants = /obj/item/clothing/under/roguetown/trou/leather/advanced
+			wrists = /obj/item/clothing/wrists/roguetown/wrappings
+			cloak = /obj/item/clothing/cloak/cape
+			head = /obj/item/clothing/head/roguetown/roguehood/white
+			shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/webs
+			armor = /obj/item/clothing/suit/roguetown/armor/plate/half/leper
+			shoes = /obj/item/clothing/shoes/roguetown/gladiator
+			belt = /obj/item/storage/belt/rogue/leather
+			r_hand = /obj/item/rogueweapon/greatsword/brokensword
+			ADD_TRAIT(H, TRAIT_NOBLE, TRAIT_GENERIC)
+			ADD_TRAIT(H, TRAIT_HEAVYARMOR, TRAIT_GENERIC)
+			ADD_TRAIT(H, TRAIT_MISSING_NOSE, TRAIT_GENERIC) //nose gone, halved stamina regen.
+			//leprosy
+			if(!H.has_status_effect(STATUS_EFFECT_SPASMS))
+				H.apply_status_effect(STATUS_EFFECT_SPASMS)
+			if(!H.has_status_effect(STATUS_EFFECT_FAKE_VIRUS))
+				H.apply_status_effect(STATUS_EFFECT_FAKE_VIRUS)
+			H.dna.species.soundpack_m = new /datum/voicepack/male/knight()
+	if(classchoice != "Leper") //those are busted enough they should not get fighting styles.
+		H.give_fightingstyle(TRUE)
```

Comment:
Is it really that busted? 
You have an innate -4 perception with muscle spasms
Your mask then add an addtional -6 or so to that
Even with the biggest perception boost statpack + race you're getting 3 perception, can you even hit anything other than centermass with that?

Reply:
No muscle spasms as long as your mask lasts

Reply:
True but its still making your perception a 2 or 3 at best in exchange for the spasms

Im not saying its not strong but it may just be unable to hit head or limbs

Reply:
-5 on mask and less perc debuff now